syntax = "proto3";

package Atlas.CoreferenceResolver;
option csharp_namespace = "Atlas.Core.Clients.Generated";

/* Request for coreference resolution. */
message CoreferenceRequest {

    // The text to analyze for coreferences
    string text = 1;
}
